templates/vanilla/packages/client/src/mud/setupNetwork.ts

```diff
@@ -1,18 +1,52 @@
+/* Definitions required to connect to a blockchain */
+
+/*
+ * The MUD client code is built on top of Viem
+ * (https://viem.sh/docs/getting-started.html).
+ * This line imports the functions we need from it.
+ */
 import { createPublicClient, fallback, webSocket, http, createWalletClient, Hex, parseEther, ClientConfig } from "viem";
 import { createFaucetService } from "@latticexyz/services/faucet";
 import { encodeEntity, syncToRecs } from "@latticexyz/store-sync/recs";
+
+/* Get the network configuration. */
 import { getNetworkConfig } from "./getNetworkConfig";
 import { world } from "./world";
 import IWorldAbi from "contracts/abi/IWorld.sol/IWorld.abi.json";
 import { createBurnerAccount, createContract, transportObserver, ContractWrite } from "@latticexyz/common";
+
+/*
+ * Use the RxJS library (https://rxjs.dev/) to create
+ * event handlers.
+ */
```

Comment:
I don't think this is necessary

Reply:
I didn't know what this was and I had to look in `package.json` and then Google for it. So I think it is useful.

Reply:
I just don't think this comment is additive (it's clear to me that we're importing from rxjs lib) and these aren't really event handlers, but observables/streams. I think Googling to learn about a library is just fine!